File: math/decimal_to_bin_oct_hex/java/DecimalToAll.java
import java.util.Scanner;

public class DecimalToAll {

  public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    int number = in.nextInt(); // get number
    System.out.println("The decimal value of " + number + " is:");
    System.out.println(Integer.toBinaryString(number) + " in binary.");
    System.out.println(Integer.toOctalString(number) + " in octal.");
    System.out.println(Integer.toHexString(number) + " in hexadecimal.");
  }
}

File: math/decimal_to_bin_oct_hex/ruby/decimal_to_all.rb
HEXA = 16
OCTA = 8
BINA = 2

puts "Enter a decimal number: "
number = gets.to_i
##*Gets* will take input from console and it will always be string.
##*To_i* will convert the string into integer.

print(number.to_s(BINA)," in binary.\n")
print(number.to_s(OCTA)," in octal.\n")
print(number.to_s(HEXA), " in hexadecimal.\n")
